runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/FaloTheBardClue.java

```diff
@@ -50,15 +54,23 @@
 @Getter
 public class FaloTheBardClue extends ClueScroll implements NpcClueScroll
 {
+	static final AnyRequirementCollection DRAGON_DEFENDER_OR_AVERNIC_DEFENDER = any("Dragon or Avernic Defender",
+		Stream.of(
+			ItemVariationMapping.getVariations(DRAGON_DEFENDER).stream(),
+			ItemVariationMapping.getVariations(AVERNIC_DEFENDER).stream())
+			.reduce(Stream::concat)
+			.orElseGet(Stream::empty)
```

Comment:
```suggestion
			.orElseGet(Stream::empty)
			.filter(itemId -> itemId != DRAGON_DEFENDER_BROKEN && itemId != AVERNIC_DEFENDER_BROKEN)
```
I haven't tested these but since we didn't handle them before, they probably don't work